src/libraries/Microsoft.PowerFx.Interpreter/ParsedExpression.cs

```diff
@@ -133,6 +135,12 @@ public async Task<FormulaValue> EvalAsync(CancellationToken cancellationToken, I
                 hasInnerServices = true;
             }
 
+            if (_features != null)
+            {
+                innerServices.AddService(_features);

```

Comment:
>             innerServices.AddService(_features); [](http://example.com/codeflow?start=3&length=49)

since interpreter only supports V1, do we need feature flags?

Reply:
I decided to leave this check because other hosts can forcibly pass `Features` without the PFxV1 flag active.

Reply:
oh, I see  - this is used to enforce these aren't executed in pre-v1 mode. cool

Reply:
But no hosts can pass Features without the PFxV1 flag active. There is only one public member of the Features class, which is Features.PowerFxV1. Everything else is internal (i.e., only Power Apps can see it, and it doesn't use the interpreter).. I don't think this is necessary.